CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- Remove expression language (#1384)
```

Comment:
We already mentioned this in https://github.com/telefonicaid/iotagent-node-lib/releases/tag/3.2.0

> Remove: remove legacy expression support (removing IOTA_DEFAULT_EXPRESSION_LANGUAGE env var and associated defaultExpressionLanguage config.js setting) (https://github.com/telefonicaid/iotagent-node-lib/issues/1340)

So I'd suggest something like this:

```suggestion
- Hardening: remove all stuff related with legacy expressions (removed in 3.2.0 but some leftovers remains in the code) (#1384)
```

Reply:
Or maybe better:

```suggestion
- Remove expressionLanguage field (as it it not longer needed, due to only one expression language is supported, after the removal of legacy expressions in 3.2.0) (#1384)
```